src/facet.scss

```diff
@@ -0,0 +1,171 @@
+
+    @include fd-rtl() {
+      margin-right: 0;
+      margin-left: $fd-image-facet-mobile-spacing;
+    }
+  }
+
+  @include fd-reset();
+
+  @include fd-flex() {
+    flex-wrap: wrap;
+  }
+
+  &__title-container {
+    @include fd-reset();
+    @include fd-flex(column);
+
+    margin-bottom: $fd-facet-title-vertical-spacing;
+  }
+
+  &__title {
+    @include fd-reset();
+
+    font-size: var(--sapFontLargeSize);
+  }
+
+  &__image {
+    @include fd-reset();
+    @include fd-flex();
+
+    @media (min-width: 0) {
+      @include image-facet-mobile-spacing();
+    }
+
+    @media (min-width: 37rem) {
+      @include facet-spacing();
+    }
+
+    // when used in object/dynamic page header
+    &--header {
+      margin-right: $fd-image-facet-header-horizontal-spacing;
+      margin-bottom: 0;
+      align-items: center;
+
+      @include fd-rtl() {
+        margin-right: 0;
+        margin-left: $fd-image-facet-header-horizontal-spacing;
+      }
+    }
+
+    // when used in object/dynamic page header in mobile
+    &--sm {
+      @include image-facet-mobile-spacing();
+    }
+  }
+
+  &__key-value {
+    @include fd-reset();
+    @include facet-spacing();
+
+    .#{$block}__object-status {
+      font-family: var(--sapFontFamily);
+      font-size: $fd-object-status-facet-font-size;
+      // font-weight: var(--sapFontHeaderWeight);  // @sapFontHeaderWeight is not part of the current theming base content and is only available as part of UI5. Currently under evaluation.
+      font-weight: normal;
+
+      & > [class*=sap-icon] {
+        font-size: $fd-object-status-facet-font-size; // of icon
+        margin-right: $fd-form-kv-facet-horizontal-spacing;
+
+        @include fd-rtl() {
+          margin-right: 0;
+          margin-left: $fd-form-kv-facet-horizontal-spacing;
+        }
+      }
+    }
+
+    .#{$block}__object-number-text, .#{$block}__object-number-unit {
+      line-height: 1;
+      padding-top: 0;
+    }
+  }
+
+  &__form-container {
+    @include fd-reset();
+    @include fd-flex(column);
+    @include facet-spacing();
+  }
+
+  &__form-label-value {
+    @include fd-reset();
+    @include fd-flex();
+
+    margin-bottom: $fd-image-form-facet-bottom-spacing;
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+  }
+
+  &__form-value {
+    @include fd-reset();
+
+    font-family: var(--sapFontFamily);
+    font-size: var(--sapFontSize);
+    font-weight: normal;
+    color: var(--sapGroup_TitleTextColor);
+
+    .#{$block}__link {
+      margin-left: $fd-form-kv-facet-horizontal-spacing;
+
+      @include fd-rtl() {
+        margin-left: 0;
+        margin-right: $fd-form-kv-facet-horizontal-spacing;
+      }
+    }
+  }
+
+  &__rating-indicator-container {
+    @include fd-reset();
+    @include facet-spacing();
+
+    display: inline-flex;
+    justify-content: flex-start;
+    align-items: flex-start;
+    flex-direction: column;
+  }
+
+  &__rating-indicator {
+    @include fd-reset();
```

Comment:
This is used like this `fd-rating-indicator fd-facet__rating-indicator` and if you reset it here, you will reset the styling coming from `fd-rating-indicator`